cli/README.md

```diff
@@ -133,10 +133,11 @@ Starts a local jamsocket dev server. You may configure the dev server with a jam
 
 ```
 USAGE
-  $ jamsocket dev [-d <value>] [-w <value>] [-p <value>] [-i]
+  $ jamsocket dev [-f <value>] [-c <value>] [-w <value>] [-p <value>] [-i]
 
 FLAGS
-  -d, --dockerfile=<value>  Path to the session backend's Dockerfile
+  -c, --context=<value>     Path to the build context for the Dockerfile (defaults to current working directory)
+  -f, --dockerfile=<value>  Path to the session backend's Dockerfile
```

Comment:
why are we switching from -d to -f?

Reply:
`-f` is used by docker commands to point to a Dockerfile. This keeps backwards compatibility with `-d` though